### DISCUSSION TITLE
Linux VM available! Works really well.

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1078] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Link: https://git.mollomm1.dev/Mollomm1/DesktopOnCodespaces
Set your codespace to four or more cores for better performance.
Also somewhat laggy on slow networks(like for example, my school network) because it works by streaming the VM.

All credit goes to Mollomm1.

UPDATE Nov 19 2024: The git subdomain of this website has been down for around the last month. Use something like BlobeVM until it comes back up.

**Comment by helloIlovegimkit** [comment]
it's pretty good and I do want to say that we should replace this with the microsoft vm because this one is fully unblocked you're even running the best os Linux

**Reply by FunnyCorgi** [reply]
also I found a better one(with minecraft and stuff) based on that: https://github.com/Blobby-Boi/BlobeVM

**Reply by FunnyCorgi** [reply]
yeah linux is really good

**Reply by ghost** [reply]
i maked by forked and improved it on https://github.com/coder0-code/BlobeVM

**Comment by S-PScripts** [comment]
wrong category i think

**Comment by schoolexploitkid** [comment]
![image](https://github.com/user-attachments/assets/2c42571c-71d9-4f0c-8193-1fc28929a12b)

**Reply by jeffrey6576i** [reply]
same for me too
